DQM/Physics/src/EwkMuLumiMonitorDQM.cc

```diff
@@ -235,11 +235,8 @@ void EwkMuLumiMonitorDQM::analyze(const Event& ev, const EventSetup&) {
       bool prescaled = false;
       const unsigned int prescaleSize = hltConfigProvider_.prescaleSize();
       for (unsigned int ps = 0; ps < prescaleSize; ps++) {
-        const unsigned int prescaleValue = hltConfigProvider_.prescaleValue(ps, trig);
-        if (prescaleValue != 1)
+        if (hltConfigProvider_.prescaleValue<double>(ps, trig) != 1)
```

Comment:
Aren't you affected by possible rounding effects of the `double` quantity here?

Reply:
See https://github.com/cms-sw/cmssw/pull/39220#discussion_r975393631.